src/citrine/_rest/ai_resource_metadata.py

```diff
@@ -29,11 +29,8 @@ class AIResourceMetadata():
 
     status = properties.Optional(properties.String(), 'status', serializable=False)
     """:Optional[str]: short description of the resource's status"""
-    status_info = properties.Optional(
-        properties.List(properties.String()),
-        'status_info',
-        serializable=False
-    )
+    status_info = properties.Optional(properties.List(properties.String()), 'status_info',
+                                      serializable=False)
```

Comment:
How'd this change happen?  Some kind of auto-linting?

Reply:
No, I probably did it when an earlier iteration made some changes to this file. Although maybe it was yelling at me, because I don't know why I'd mess with the formatting with no other change?